File: 1945-sum-of-digits-of-string-after-convert/1945-sum-of-digits-of-string-after-convert.java
class Solution {
    public int getLucky(String s, int k) {
        String intStr="";
        for(int i=0;i<s.length();i++) intStr+=(s.charAt(i)-'a'+1);
        int sum=0;
        while(k>0){
            sum=0;
            for(int i=0;i<intStr.length();i++){
                sum+=(intStr.charAt(i)-'0');
            }
            intStr= String.valueOf(sum);
            k--;
        }
        return sum;
    }
}

File: 1945-sum-of-digits-of-string-after-convert/1945-sum-of-digits-of-string-after-convert.cpp
class Solution {
public:
    int getLucky(string s, int k) {
        string intStr="";
        for(auto &it:s) intStr+=to_string(it-'a'+1);
        int sum=0;
        while(k--){
            sum=0;
            for(auto &it:intStr){
                sum+=(it-'0');
            }
            intStr=to_string(sum);
        }
        return sum;
    }
};